airflow/www/views.py

```diff
@@ -529,7 +529,7 @@ def code(self, session=None):
             html_code = highlight(
                 code, lexers.PythonLexer(), HtmlFormatter(linenos=True))
         except OSError as e:
-            html_code = str(e)
+            html_code = 'Failed to load file.<br/> Details: ' + str(e)
```

Comment:
```suggestion
            html_code = 'Failed to load file.<br/> Details: ' + escape(str(e))
```

Should use https://flask.palletsprojects.com/en/1.1.x/api/?highlight=safe#flask.escape